### PR TITLE
 add: `onlyoffice-desktopeditors-deb`

### PR DESCRIPTION
**Pull request by soumyaDghosh**

### CODE REVIEW

**Comment by soumyatheman** [comment]
Please check this, pacstall for some reason not working in my machine, I can't use pacstall -I or -Up or -U. That's also a reason I guess, my merge is failing. please check

**Comment by Henryws** [comment]
Heh? Run `pacstall -V`

**Comment by soumyatheman** [comment]
2.0.1 Mellow

**Comment by Henryws** [comment]
Can you upgrade with `pacstall -U pacstall master`?

**Comment by soumyatheman** [comment]
I actually just installed the update, but the issue is, pacstall is working very slow. After typing `pacstall -U pacstall master`, I waited for almost 5 minutes and nothing happened, so I installed using deb file, but now I typed, `pacstall -Up`, and yet nothing seems to be showing up

**Comment by soumyatheman** [comment]
[Screencast from 2022-12-06 00-08-57.webm](https://user-images.githubusercontent.com/72045785/205716821-e1373fa8-ad23-4323-b94f-17bd3dba5d6c.webm)

Please check this

**Comment by soumyatheman** [comment]
headset-deb
linked-deb
ms-teams-deb
onlyoffice-desktopeditors-deb
songrec-deb
teams-for-linux-deb
telegram-bin
webcord-deb
zoom-deb

**Comment by Henryws** [comment]
Edit `/usr/share/pacstall/scripts/upgrade.sh`, and on the 2nd line, add the text `set +x`, then rerun the upgrade.

**Comment by soumyatheman** [comment]
Same

`[+] INFO: Checking for updates
[!] ERROR: Packagelist not found
[!] ERROR: Packagelist not found
[⠿] Suggested solution is:
    | Confirm that 'https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packagelist/packagelist' exists
[⠿] Suggested solution is:
    | Confirm that 'https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packagelist/packagelist' exists
[!] ERROR: Packagelist not found
[!] ERROR: Packagelist not found
[⠿] Suggested solution is:
    | Confirm that 'https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packagelist/packagelist' exists
[⠿] Suggested solution is:
    | Confirm that 'https://raw.githubusercontent.com/pacstall/pacstall-programs/master/packagelist/packagelist' exists
[*] WARNING: Skipping repo pacstall/pacstall-programs
[*] WARNING: Skipping repo pacstall/pacstall-programs
[*] WARNING: Skipping repo pacstall/pacstall-programs
[*] WARNING: You can remove or fix the URL by editing /usr/share/pacstall/repo/pacstallrepo.txt
[*] WARNING: You can remove or fix the URL by editing /usr/share/pacstall/repo/pacstallrepo.txt
[*] WARNING: You can remove or fix the URL by editing /usr/share/pacstall/repo/pacstallrepo.txt
[*] WARNING: Skipping repo pacstall/pacstall-programs
[*] WARNING: You can remove or fix the URL by editing /usr/share/pacstall/repo/pacstallrepo.txt
[!] ERROR: There is no package with the name telegram-bin
[!] ERROR: There is no package with the name teams-for-linux-deb
[!] ERROR: There is no package with the name headset-deb
[!] ERROR: There is no package with the name ms-teams-deb
[+] INFO: Nothing to upgrade`

**Comment by Henryws** [comment]
Send the contents of `/usr/share/pacstall/repo/pacstallrepo.txt`

**Comment by soumyatheman** [comment]
https://raw.githubusercontent.com/pacstall/pacstall-programs/master

**Comment by soumyatheman** [comment]
That link is actually not working

**Comment by Henryws** [comment]
Try reinstalling pacstall.

**Comment by soumyatheman** [comment]
Uninstalling command also seems to be taking a long time

**Comment by Henryws** [comment]
What command are you running?

**Comment by Henryws** [comment]
Ok that's your problem then.

**Comment by soumyatheman** [comment]
> Ok that's your problem then.

ISP issues?

**Comment by Henryws** [comment]
Maybe.

**Comment by soumyatheman** [comment]
But also, why these checks aren't completing?

**Comment by Henryws** [comment]
I don't know.

**Comment by soumyatheman** [comment]
BTW that's fixed, it's ISP issue

**Comment by soumyatheman** [comment]
> I don't know.
> 
> EDIT: it's probably a debconf issue.

Any fix?

**Comment by soumyatheman** [comment]
> BTW that's fixed, it's ISP issue

Pacstall working properly here now

**Comment by soumyatheman** [comment]
Thanks

File: packages/onlyoffice-desktopeditors-deb/onlyoffice-desktopeditors-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name="onlyoffice-desktopeditors-deb"
+gives="onlyoffice-desktopeditors"
+version="7.2.1"
+replace="${gives}"
+url="https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v${version}/onlyoffice-desktopeditors_amd64.deb"
+description="ONLYOFFICE Desktop Editors is a free office suite that combines text, spreadsheet and presentation editors."
```
Comment (Henryws) [inline packages/onlyoffice-desktopeditors-deb/onlyoffice-desktopeditors-deb.pacscript:6]:
```suggestion
description="ONLYOFFICE Desktop Editors is a free office suite that combines text, spreadsheet and presentation editors"
```

**Pull request merged**